python/generate/templates/rest.mustache

```diff
@@ -15,6 +15,7 @@ import certifi
 import six
 from six.moves.urllib.parse import urlencode
 import urllib3
+import simplejson as json
```

Comment:
Do we need to remove the `json` import above then? We have two imports with the same name here.